scripts/generate.mjs

```diff
@@ -0,0 +1,29 @@
+import { Octokit } from 'octokit';
+import * as fs from 'node:fs/promises';
+
+const gh = new Octokit();
+
+const searchResults = gh.paginate.iterator(gh.rest.search.repos, {
+    q: 'titanium in:topics language:objc language:swift language:java language:kotlin language:javascript'
+});
+
+const repositories = [];
+for await (const {data} of searchResults) {
+    repositories.push(...data.map(mapData));
+}
+
+const out = new URL('../data.json', import.meta.url);
+await fs.writeFile(out, JSON.stringify(repositories, undefined, '\t'));
+
+function mapData(data) {
+    return {
```

Comment:
imo, would be neat include things like platform in this data, but that's not really possible to infer from a simple GitHub search AFAICT because a repo can only have a single language. It might be best to not actually have this be a generated file, but instead be a curated list so that things like platform can be stated easily

Reply:
It would be cool if we could somehow parse the `timodule.xml` (and `manifest`) to get the module info. If the repo doesn't have one of those two files, then it's not a Titanium Module, right?

Reply:
I guess 7d9e359bddc991f4c6cb6d15d27b356969454440 kinda made my comment here outdated, currently it's going off whether a `android`, `ios`, or `iphone` folders exist, I guess extending that to `<platform>/timodule.xml` over folder names is safer.

But honestly I think that automatically generating the file it isn't the way to go, I think having some form of curated list is best